### ISSUE TITLE
Unable to delete empty line when pressing backspace in multiple choice title

### ISSUE THREAD

**Issue by juicemind-dev** [JuiceMind/JuiceMind#147]

**Comment by kylejramstad** [comment]
Similar issue when trying to remove the last bullet point in a list. It won't delete an empty last bullet point.
Example

- Things
- Stuff
-

That last bullet cannot be deleted with a backspace

**Comment by juicemind-dev** [comment]
Yes, thanks for noting that @kylejramstad . We've elevated this to Customer Request level so that it gets done faster.
